### DISCUSSION TITLE
Soft prompts

### DISCUSSION THREAD

**Discussion by evstarshov** [LostRuins/koboldcpp#154] [Q&A]
Is there any way to use soft prompts in this project? 🤔

**Comment by LostRuins** [comment]
Sorry, soft prompts are not supported in this project, however if it is a LLAMA model, you can use LORAs which are similar.

**Reply by VL4DST3R** [reply]
Will they ever be supported? From what I understand the full UI already does? They seem to be a lot more flexible for the end-user to create.
